api/src/models/homepage.model.js

```diff
@@ -11,13 +11,11 @@ const homepageSchema = new mongoose.Schema(
       ref: 'team',
       required: true,
     },
-    // `aboutUs` stores a list of strings.
-    // Each string in list is a paragraph.
-    aboutUs: [
-      {
-        type: String,
-      },
-    ],
+    // `aboutUs` stores a HTML string
+    aboutUs:
```

Comment:
Could you update the trigger in the team model as well? I believe we’re defaulting to an empty array, which should now be an empty string. You can replace the empty array assertion with empty string in the test as well